cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -308,14 +342,28 @@ instance IsEffectuator LiveViewBackend Text where
                             -- Check for unexpected thunks
                             checkForUnexpectedThunks ["Net.IpExt:InOctets LiveViewBackend"] lvs
 
+                            peerStates <- fmap tuple3pop . fromMaybe mempty <$>
+                              sequence (atomically . (>>= traverse Net.readFetchClientState) . Net.readFetchClientsStateVars . getFetchClientRegistry . getNodeKernel <$> lvsNodeKernel lvs)
+                            candidates <- fromMaybe mempty <$>
+                              sequence (atomically . getCandidates . getNodeCandidates . getNodeKernel <$> lvsNodeKernel lvs)
+
                             return $ lvs { lvsNetworkUsageInCurr = currentNetRate
                                          , lvsNetworkUsageInPerc = (currentNetRate / (maxNetRate / 100.0)) / 100.0
                                          , lvsNetworkUsageInLast = inBytes
                                          , lvsNetworkUsageInNs   = currentTimeInNs
                                          , lvsNetworkUsageInMax  = maxNetRate
                                          , lvsUpTime             = diffUTCTime (tstamp meta) (lvsStartTime lvs)
+                                         , lvsPeers = Map.elems . flip Map.mapMaybeWithKey candidates $
```

Comment:
You might want to pull this calculation out into a supporting function and put it near the newtype above - then it could be clearly documented that this is dependent on details of the wrapped data structure - just for longer term maintenance

Reply:
Done, good point!